### PR TITLE
Nsetm 2101 split files

### PR DESCRIPTION
**Pull request by edasubert**

### CODE REVIEW

File: bluepysnap/edges/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""Edges and EdgePopulation objects."""
+
+from .edge_population import EdgePopulation
+from .edges import Edges
```
Comment (GianlucaFicarelli) [inline bluepysnap/edges/__init__.py:4]: These relative imports should be converted to absolute imports, for consistency with the other imports in the project and for better readability.

File: bluepysnap/edges/edges.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+"""Edge population access."""
```
Comment (GianlucaFicarelli) [inline bluepysnap/edges/edges.py:18]: Minor
```suggestion
"""Edges access."""
```

File: bluepysnap/nodes/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+"""Nodes and NodePopulation objects."""
+
+from .node_population import NodePopulation
+from .nodes import Nodes
```
Comment (GianlucaFicarelli) [inline bluepysnap/nodes/__init__.py:4]: Same comment as for the edges.

File: bluepysnap/nodes/nodes.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+"""Node population access."""
```
Comment (GianlucaFicarelli) [inline bluepysnap/nodes/nodes.py:18]: Minor
```suggestion
"""Nodes access."""
```

**Review by GianlucaFicarelli — COMMENTED** [review]
I see that the history of commits is quite confusing. Do you mind squashing and rebasing?
I suppose that the commits will be automatically squashed and rebased before merging the PR, but I feel more confident to review the changes if it's done before.

**Review by joni-herttuainen — COMMENTED** [review]
Blame-wise everything seems to be in order. It seems to me that you addressed all the things that @GianlucaFicarelli pointed out but I let him double-check and approve.

**Review by GianlucaFicarelli — APPROVED** [review]

**Pull request merged**